<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>740</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="pushButton_halal">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>650</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Халяль</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_haram">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>650</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Харам</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-40</x>
      <y>110</y>
      <width>800</width>
      <height>505</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>321</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Начать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>480</width>
      <height>270</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_prosh">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>прошло времени</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ost">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>осталось времени</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>25</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <property name="autoScrollMargin">
     <number>15</number>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <property name="columnCount">
     <number>5</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>1</string>
     </property>
    </column>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <zorder>pushButton_halal</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>lcdNumber</zorder>
   <zorder>pushButton_start</zorder>
   <zorder>label_3</zorder>
   <zorder>lcdNumber_2</zorder>
   <zorder>label_prosh</zorder>
   <zorder>label_ost</zorder>
   <zorder>tableWidget</zorder>
   <zorder>pushButton_haram</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tableWidget</tabstop>
  <tabstop>pushButton_start</tabstop>
  <tabstop>pushButton_halal</tabstop>
  <tabstop>pushButton_haram</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
